### ISSUE TITLE
[A11ySev2] [Programmatic access -Dashboards with Grafana - New Dashboard- Add Action]: Confirmation message label is not associated programmatically for "Are you sure you want to..?" edit field.

### ISSUE THREAD

**Issue by LinglingTong** [grafana/grafana#118760]
## Steps to reproduce

Repro Steps: 
1. Open 'Azure Monitor dashboards with Grafana' in Azure Portal
2. "Monitor | Dashboards with Grafana | Gallery" screen appears with already Dashboards with Grafana tab selected.  
3. Navigate to the all the controls available on Dashboards with Grafana blade like tool bar , controls under All Dashboards tab.  
4. Navigate New menu present in the tool bar and select New Dashboard.  
5. Navigate to "Add visualization" button and hit ENTER key. "Select data source" popup will appear.  
6. Navigate to the TestData list item select it.   
7. Now navigate to all the dialogs which is appears after invoking the add buttons like add link , add action , add value mappings , etc. under Visualization pane. 
8. Observe that screen reader is not announcing the label for “Are you sure you want to..?” edit field. 

**Actual Result:**
Screen readers fail to announce a label for the "Are you sure you want to..?" edit field, leaving its purpose unclear to users with visual impairments.  
**Observation**:  
The "Are you sure you want to..?" edit field under the "Add Action" tab does not have an accessible name programmatically defined using attributes such as aria-label, aria-labelledby, or title. As a result, screen readers announces "Are you sure you want to..? Edit blank" for the control.  
**Issue1:** Same issue reproduces for the "Key/Value" edit fields under add action tab by invoking add action button on the right side of the page.  
**Navigation**: Open https://portal.azure.com/#view/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/~/AzureGrafanaReactView  > Select New button button > Invoke New dashboard control> Select “ Add visualization” button > Select testdata list item> Navigate to all controls under  “Data links and actions”> Select add action button and invoke it >Verify that the screen reader announces the label for “key/ Value” edit fields under the “Variables”, “Query Parameters” and “headers” heading on the “Add action “ tab. 
**Issue2**: Same issue repro combo box and edit field under the Legend section after activating the  'Symlog' radio button.  
**Navigation**: Open https://portal.azure.com/#view/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/~/AzureGrafanaReactView  > Select New button button > Invoke New dashboard control> Select “ Add visualization” button > Select testdata list item> Navigate all the controls under ‘Axis’ section > Select ‘Symlog’ radio button under scale section > Observe that the screen reader does not announce label for the Log base combo box and Linear threshold edit field.  

**Expected Result:**
The "Are you sure you want to..?" edit field should have an accessible name that is programmatically defined and screen reader should announce "Confirmation message, Provide a descriptive prompt to confirm or cancel the option, Are you sure you want to..?' Edit blank."  This can be achieved by adding attributes such as aria-label, aria-labelledby, or title.

**Relevant WCAG Criteria:** [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) (Level A)

## Environment

- **Grafana version:** 12.2.0
- **Data source type & version:**
- **User OS & Browser:** Version 25H2 (OS Build 26200.7093) / Chrome Version 142.0.7444.177 (Official Build) (64-bit)
- **Others:**

**Comment by LinglingTong** [comment]
<img width="1919" height="1082" alt="Image" src="https://github.com/user-attachments/assets/a0aae542-de3b-4a5e-a86d-534a0d0534ec" />
<img width="1919" height="1084" alt="Image" src="https://github.com/user-attachments/assets/e594b398-359c-4d69-8bd7-1cb89710e9cc" />
<img width="1916" height="1090" alt="Image" src="https://github.com/user-attachments/assets/6a2816c2-7e23-462a-8395-87beaca9657f" />

**Comment by v-kavyagupta** [comment]
We verified this issue on the URL:http://localhost:3000/dashboard/new?orgId=1&from=now-7d&to=now&timezone=browser&var-adhoc0=&dtab=new-tab&editPanel=2
Issue is still repro, Confirmation message label is not associated programmatically for "Are you sure you want to..?" edit field.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/2c9017cf-eaf8-4219-9195-687114b1f726" />
